com.endlessm.apps.Sdk.json.in.in

```diff
@@ -501,6 +501,41 @@
                 "tar -C /usr/share/eos-hotdoc-theme -xJf eos-hotdoc-theme-0.0.0.tar.xz"
             ]
         },
+        {
+            "name": "bison",
+            "cleanup-platform": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "url": "http://ftp.gnu.org/gnu/bison/bison-3.0.4.tar.gz",
+                    "sha256": "b67fd2daae7a64b5ba862c66c07c1addb9e6b1b05c5f2049392cfd8a2172952e",
+                    "type": "archive"
+                }
+            ]
+        },
+        {
+            "name": "flex",
+            "cleanup-platform": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "url": "https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz",
+                    "sha256": "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995",
+                    "type": "archive"
+                }
+            ]
+        },
+        {
+            "name": "mustache-c",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/x86-64/mustache-c.git"
```

Comment:
This is kind of problematic: building from Git should only be reserved to components we control, since we also control the release schedule. Third party modules should either be built from a Git tag or from a tarball.

Reply:
Fair enough. This dependency doesn't really have a release schedule to speak of, but I suppose we can just pin it to a certain commit (or create our own fork and pull from that).

Reply:
Yeah, please pin it to a specific commit you know working, if you can't use a tag or tarball.